docs/operational_guide/topology.md

```diff
@@ -0,0 +1,173 @@
+# Topology
```

Comment:
Maybe just call this Placement to stay consistent with code and say that it's the cluster topology in a note?